.gitlab-ci.yml

```diff
@@ -349,21 +387,21 @@ build_puppy_agent-deb_x64_arm:
     - source /root/.bashrc && conda activate ddpy3
     - inv deps --verbose --dep-vendor-only --no-checks
   script:
-    - GOOS=linux GOARCH=arm inv -e agent.build --puppy
+    - inv -e agent.build --puppy
+    - $S3_CP_CMD $SRC_PATH/$AGENT_BINARIES_DIR/agent $S3_ARTIFACTS_URI/puppy/agent
 
-# build dogstatsd for deb-x64
-build_dogstatsd-deb_x64:
+# build puppy agent for ARM, to make sure the build is not broken because of build targets
+build_puppy_agent-deb_x64_arm:
   stage: binary_build
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
```

Comment:
Can you fix this not actually building for ARM? Or are you going to do that in a different PR?

Reply:
I thought doing it in another PR